pkg/controller/idler/idler_controller.go

```diff
@@ -165,9 +168,11 @@ func (r *ReconcileIdler) ensureIdling(logger logr.Logger, idler *toolchainv1alph
 // and scales the owner down to zero and returns "true".
 // Otherwise returns "false".
 func (r *ReconcileIdler) scaleControllerToZero(logger logr.Logger, meta metav1.ObjectMeta) (bool, error) {
+	logger.Info("Scaling controller to zero", "name", meta.Name)
 	owners := meta.GetOwnerReferences()
 	for _, owner := range owners {
 		if owner.Controller != nil && *owner.Controller {
+			log.Info("owner", "owner", owner.Kind)
```

Comment:
Do we still need this log? I think that it was added just for troubleshooting. If we need it, then please fix the content to log something more meaningful.

Reply:
Yeah... I added most of these logs just for debugging. Not sure if we want to be so verbose in our idler logs.

Reply:
removed in https://github.com/codeready-toolchain/member-operator/pull/205/commits/b587338166eb5aa34e9a3149cd7c90c321cdc27d